std/stdio.d

```diff
@@ -17,6 +17,7 @@ Authors:   $(WEB digitalmars.com, Walter Bright),
 module std.stdio;
 
 public import core.stdc.stdio;
+public import std.exception : ErrnoException;
```

Comment:
No thanks - rather there should be common supertype for OS-agnostic system failures. Notably windows is no errorno-based.